src/fprime/util/cli.py

```diff
@@ -242,7 +242,8 @@ def validate(parsed, unknown):
     # Build type only for generate, jobs only for non-generate
     elif parsed.command in [target.mnemonic for target in Target.get_all_targets()]:
         parsed.settings = None  # Force to load from cache if possible
-        make_args["--jobs"] = 1 if parsed.jobs <= 0 else parsed.jobs
+        if parsed.jobs:
```

Comment:
`parsed.jobs is None` otherwise `0` will skip this check and miss the `make_args["--jobs"] = 1 if parsed.jobs <= 0 else parsed.jobs` correction

Reply:
Now that I think about it, we should have

```
if parsed.jobs is None or parsed.jobs <=0:
    make_args["--jobs"] = parsed.jobs
....

Reply:
That way --jobs is only set when valid, otherwise default.

Reply:
Updated